src/vs/workbench/services/search/common/search.ts

```diff
@@ -311,6 +311,17 @@ export class OneLineRange extends SearchRange {
 	}
 }
 
+export const SearchSortOrderConfiguration = {
```

Comment:
These two can be unified into a single `const enum`. Following that, the `== 'modified'`-style checks elsewhere should be changed to `== SearchSortOrder.Modified`. Also we aren't super consistent about `CAPS_CASE` vs. `PascalCase` for `const enums` in elsewhere in vscode, but I think here `PascalCase` feels better. Up to you.